components/script/dom/element.rs

```diff
@@ -2816,6 +2829,11 @@ impl Element {
         self.set_disabled_state(has_disabled_attrib);
         self.set_enabled_state(!has_disabled_attrib);
     }
+
+    pub fn get_node(&self) -> &Node {
```

Comment:
This looks like an extraneous function. For places where you are supplied with an `Element`, you can call `element.upcast::<Node>()` to get a `Root<Node>`.